drivers/volume/portworx/portworx.go

```diff
@@ -227,6 +276,17 @@ func (p *portworx) mapNodeStatus(status api.Status) storkvolume.NodeStatus {
 	}
 }
 
+func (p *portworx) getNodeLabels(hostname string) (map[string]string, error) {
+	obj, exists, err := p.store.GetByKey(hostname)
```

Comment:
Does this assume that the k8s node name is always the hostname?

Reply:
Yes, that was the assumption previously too when calling k8s.Instance().GetLabelsOnNode()